latch/ldata/transfer/download.py

```diff
@@ -69,14 +71,11 @@ def download(
         headers={"Authorization": get_auth_header()},
         json={"path": normalized},
     )
-
     if res.status_code != 200:
-        click.secho(
+        raise HTTPException(
```

Comment:
you are expected to raise the correct exception subclass based on the code itself